staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/apiserver.go

```diff
@@ -45,6 +45,7 @@ import (
 	_ "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	_ "k8s.io/apiextensions-apiserver/pkg/client/informers/externalversions"
 	_ "k8s.io/apiextensions-apiserver/pkg/client/informers/internalversion"
+	"k8s.io/apiserver/pkg/util/webhook"
```

Comment:
import order

Reply:
Fixed and not the place to discuss it, but honestly I don't see any benefit with these import grouping when tools can auto import/order things in one list.

Reply:
These underscore imports are special. Init funcs....